balboa-kafka/src/main/scala/com/socrata/balboa/kafka/BalboaConsumerGroup.scala

```diff
@@ -0,0 +1,55 @@
+package com.socrata.balboa.kafka
+
+import java.util.concurrent.{Executors, ExecutorService}
+
+import com.socrata.balboa.kafka.consumer.TestConsumer
+import kafka.consumer.{KafkaStream, ConsumerConnector, Consumer, ConsumerConfig}
+
+import scala.collection.Map
+import scala.collection.immutable.HashMap
+
+/**
+ * Created by Michael Hotan, michael.hotan@socrata.com
+ *
+ * ADT that manages the connection with zookeeper nodes and consumption of Kafka Messages.  ADT attempts to utilize a
+ * pure functional
+ *
+ * Note: Most likely won't
+ */
+sealed trait BalboaConsumerGroup
+case class ::(consumer: ConsumerConnector, topic: String) extends BalboaConsumerGroup
+
+/**
+ * `BalboaKafka` Companion object
+ * */
+object BalboaConsumerGroup {
+
+  /**
+   * Initialize a Balboa Kafka Consumer Service using a set of configuration
+   *
+   * @param consumerConfig Consumer configurations
+   * @return Balboa Kafka instance
+   */
+  def init(consumerConfig: ConsumerConfig, topic: String): BalboaConsumerGroup = {
+    (consumerConfig, topic) match {
+      case _=> ::(Consumer.create(consumerConfig), topic)
+    }
+  }
+
+  /**
```

Comment:
My tendency with javadocs is to leave out obvious bits of information. For instance; clearly numThreads is the number of threads for this consumer. I don't think we need a @param entry for it. You also leave off the consumer group, which is probably fine... but this is an example where a more complicated parameter is not documented, where as the obvious one is.